plugins/lookup/passwordstore.py

```diff
@@ -60,19 +60,51 @@
         description: use alphanumeric characters.
         type: bool
         default: 'no'
+      missing:
+        description:
+          - List of preference about what to do if the password file is missing.
+          - If I(create=true), the value for this option is ignored and assumed to be C(create).
+          - If set to C(error), the lookup will error out if the passname does not exist.
+          - If set to C(create), the passname will be created with the provided length I(length) if it does not exist.
+          - If set to C(empty) or C(warn), will return an empty string in case the passname does not exist.
```

Comment:
```suggestion
          - If set to C(empty) or C(warn), will return C(none) in case the passname does not exist.
```
Isn't it `none` now?

Reply:
I that so? Seems to me, that it is interpreted as an empty string. Or am I missing something?
```
TASK [debug] *******************************************************************************
ok: [localhost] => {
    "lookup('passwordstore', 'test-missing-pass missing=empty')": ""
}
```

Reply:
This should document what the plugin returns, not (only) what the `lookup` magic does with its result. People might be using `query` instead of `lookup`.

Reply:
Ah, so the lookup is the place where the magic comes from.